src/components/outline-nav/types.ts

```diff
@@ -38,12 +38,3 @@ export type OutlineLinkItem = OutlineLinkProps | OutlineLinkWithNestingProps
 export interface OutlineNavProps {
 	items: OutlineLinkItem[]
 }
-
-/**
- * Heading type, for compatibility with existing conventions.
- */
-export interface OutlineHeading {
```

Comment:
I've renamed this type `AnchorLinksPluginHeading` and moved it into `lib/docs/outline-items-from-headings.ts`, since I needed this type in a few places!